includes/User.php

```diff
@@ -2131,7 +2108,9 @@ public function setInternalPassword( $str ) {
 		$this->load();
 		$this->setToken();
 
-		if( $str === null ) {
+		// @deprecated
+		// @todo remove local storing of password
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)